### ISSUE TITLE
List repos for a user or organization

### ISSUE THREAD

**Issue by gnorsilva** [spring-projects/spring-social-github#13]
You can currently get a repo's details, but that assumes you know what repos are available in an organisation.  I'd like to see a way to get a list of all the repos available for a user/org.

**Comment by SwapnaliShrikhande** [comment]
Well, we can get list of all the public repos for a user:
curl -H "Authorization: token <token_id>" https://api.github.com/users/<user_name>/repos
Correct me if I am wrong!

**Comment by mloza** [comment]
Yes, but this issue is about lack of support this in spring-social-github library, and my pull request is solving it.

**Comment by alighm** [comment]
Why hasn't this been merged yet??!!!

**Comment by michalloza** [comment]
I think repo maintainer abandoned this, the last commit is from 2 years ago 😢
